pkg/reconciler/v1alpha1/subscription/subscription.go

```diff
@@ -204,50 +213,61 @@ func isNilOrEmptyReply(reply *v1alpha1.ReplyStrategy) bool {
 }
 
 // updateStatus may in fact update the subscription's finalizers in addition to the status
-func (r *reconciler) updateStatus(ctx context.Context, subscription *v1alpha1.Subscription) (*v1alpha1.Subscription, error) {
-	objectKey := client.ObjectKey{Namespace: subscription.Namespace, Name: subscription.Name}
-	latestSubscription := &v1alpha1.Subscription{}
-
-	if err := r.client.Get(ctx, objectKey, latestSubscription); err != nil {
+func (r *Reconciler) updateStatus(ctx context.Context, desired *v1alpha1.Subscription) (*v1alpha1.Subscription, error) {
```

Comment:
I think this comment is not true anymore? It seems to only update the Status directly?

Reply:
fixed.